src/main/java/nl/hsac/fitnesse/fixture/util/selenium/SeleniumHelper.java

```diff
@@ -549,6 +549,47 @@ public void hoverOver(WebElement element) {
         getActions().moveToElement(element).perform();
     }
 
+    /**
+     * Simulates clicking on offset from center place on the supplied element
+     * @param element element to click on with ofset
+     * @param xOffset horizontal integer offset from center
+     * @param yOffset vertical integer offset from center
+     */
+    public void clickAtXoffsetAndYoffset(WebElement element, Integer xOffset, Integer yOffset) {
+        getActions().moveToElement(element, xOffset, yOffset).click().build().perform();
+    }
+
+    /**
+     * Simulates right clicking on offset from center place on the supplied element
+     * @param element element to click on with ofset
+     * @param xOffset horizontal integer offset from center
+     * @param yOffset vertical integer offset from center
+     */
+    public void rightClickAtXoffsetAndYoffset(WebElement element, Integer xOffset, Integer yOffset) {
+        getActions().moveToElement(element, xOffset, yOffset).contextClick().build().perform();
+    }
+
+    /**
+     * Simulates double clicking on offset from center place on the supplied element
+     * @param element element to click on with ofset
+     * @param xOffset horizontal integer offset from center
+     * @param yOffset vertical integer offset from center
+     */
+    public void doubleClickAtXoffsetAndYoffset(WebElement element, Integer xOffset, Integer yOffset) {
+        getActions().moveToElement(element, xOffset, yOffset).doubleClick().build().perform();
```

Comment:
please create a method to to provide `getActions().moveToElement(element, xOffset, yOffset)` used multiple times

Reply:
Done